### PR TITLE
Devel: adds input file to generate synthetic data

### LINKED ISSUE #9: Generate synthetic time histories
The next step for this methodology is to use ``RAVEN`` to generate synthetic time histories (training data for an reduced order model). 

Read the section in the user guide on ``trainARMA``

This issue can be closed when a ``RAVEN`` input file for generating typical time histories has been created.

### PR DESCRIPTION
**Pull request by samgdotson**
Closes #9 
This PR adds a ``RAVEN`` input filet that generates synthetic data. It updates the associated jupyter notebook and adds a figure comparing the typical history with a synthetic one.

### CODE REVIEW

File: data_processing/tests/test_data_funcs.py [hunk 1]
```diff
@@ -78,3 +84,37 @@ def test_to_float3():
...
+    assert increasing and same_stats
+
+
+def test_make_increasing2():
+    """
+    Test case 2: A data series with repeated values is passed.
+    """
+
+    inc_data = make_increasing(repeat_vals, sort=False, strict=True)
+    print(inc_data)
```
Comment (robfairh) [inline data_processing/tests/test_data_funcs.py]:
```suggestion

```

File: data_processing/tests/test_data_funcs.py [hunk 2]
```diff
@@ -78,3 +84,37 @@ def test_to_float3():
...
+    print(repeat_vals)
```
Comment (robfairh) [inline data_processing/tests/test_data_funcs.py]:
```suggestion

```

File: data_processing/tests/test_data_funcs.py [hunk 3]
```diff
@@ -78,3 +84,37 @@ def test_to_float3():
...
+
+def test_make_increasing1():
+    """
+    Test case 1: A data series that is already increasing is passed.
+    """
+    inc_data = make_increasing(single_digits, sort=False, strict=True)
+    increasing = all(i < j for i, j in zip(inc_data, inc_data[1:]))
+    old_mean = single_digits.mean()
+    new_mean = inc_data.mean()
+    same_stats = abs(old_mean - new_mean) < 0.1
```
Comment (robfairh) [inline data_processing/tests/test_data_funcs.py]:
```suggestion
    same_stats = abs(old_mean - new_mean)/abs(old_mean) < 0.01
```

File: data_processing/tests/test_data_funcs.py [hunk 4]
```diff
@@ -78,3 +84,37 @@ def test_to_float3():
...
+    Test case 2: A data series with repeated values is passed.
+    """
+
+    inc_data = make_increasing(repeat_vals, sort=False, strict=True)
+    print(inc_data)
+    print(repeat_vals)
+    increasing = all(i < j for i, j in zip(inc_data, inc_data[1:]))
+    old_mean = np.array(repeat_vals).mean()
+    new_mean = np.array(inc_data).mean()
+    same_stats = abs(old_mean - new_mean) < 0.1
```
Comment (robfairh) [inline data_processing/tests/test_data_funcs.py]:
```suggestion
    same_stats = abs(old_mean - new_mean)/abs(old_mean) < 0.01
```

**Review by robfairh — CHANGES_REQUESTED** [review]
See a few minor suggestions

**Comment by samgdotson** [comment]
Changes to function tests have been implemented.

**Pull request merged**